src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -38,12 +38,44 @@ class WorkspaceInitialPage extends React.Component {
     constructor(props) {
         super(props);
 
-        const policy = this.props.policy;
         this.openEditor = this.openEditor.bind(this);
         this.toggleDeleteModal = this.toggleDeleteModal.bind(this);
         this.confirmDeleteAndHideModal = this.confirmDeleteAndHideModal.bind(this);
 
-        this.menuItems = [
+        this.state = {
+            isDeleteModalOpen: false,
+        };
+    }
+
+    /**
+     * Open Workspace Editor
+     */
+    openEditor() { Navigation.navigate(ROUTES.getWorkspaceSettingsRoute(this.props.policy.id)); }
```

Comment:
minor comment, i believe we prefer multiline methods over single line ones so can you change this to,
```
openEditor() { 
    Navigation.navigate(ROUTES.getWorkspaceSettingsRoute(this.props.policy.id)); 
}
```
so that its consistent with the rest of our codebase.

Reply:
Done